config/example/config.txt

```diff
@@ -190,6 +190,9 @@ GUEST_JOBBAN
 ## Uncomment this to stop people connecting to your server without a registered ckey. (i.e. guest-* are all blocked from connecting)
 GUEST_BAN
 
+## above this player count threshold, never-before-seen players are blocked from connecting
+PANIC_BUNKER_THRESHOLD 150
```

Comment:
The configuration variable `panic_bunker_threshold` is set to a default of 150:
https://github.com/ParadiseSS13/Paradise/blob/4fffcd676d43caaaaaf4ac9bc20dce0f22728101/code/controllers/configuration.dm#L60

And it is overridden by the configuration only if it is specified:
https://github.com/ParadiseSS13/Paradise/blob/4fffcd676d43caaaaaf4ac9bc20dce0f22728101/code/controllers/configuration.dm#L507

I think if this line is missing from `config.txt` it will default to 150, which is probably worth noting in the comment. I don't think the game will break?